src/defs_version.h

```diff
@@ -54,5 +54,5 @@
 
     // READ ABOVE CAREFULLY BEFORE CHANGING!
     // USE 3 DIGITS VERSION ONLY /!\ _
-    #define MIXXX_VERSION "2.2.0-alpha-pre"
+    #define MIXXX_VERSION "2.2.0-beta"
```

Comment:
beta1 please

Reply:
What difference does that make?

Reply:
It lets us release a beta2 and so on without conflicts/problems especially in Debian packages.

Reply:
But we don't have "beta1" or "beta2", we continuously make new beta builds on each commit.

Reply:
That reminds me, use `~beta1` because hyphen is a higher-priority character in Debian packaging.

Reply:
You're talking about 2.2 branch "nightlies." Please do not confuse that with a beta release!!

Reply:
I don't know what you mean. We do not have discrete beta releases.

Reply:
Since when?! Talk in Zulip.

Reply:
I noticed there is a "-" rather than a "~". Is the "~" really necessary? `dpkg --compare-versions 2.2.0-alpha lt 2.2.0-beta` shows that the beta is still considered newer than the "alpha-pre" builds.

Reply:
According to https://github.com/mixxxdj/mixxx/blob/2.2/src/SConscript#L163 the `-` is correct.

Reply:
Okay, thanks for checking.

Reply:
Looks like it does the dash-to-tilde replacement at package build time automatically. Cool.